### DISCUSSION TITLE
PDF/A

### DISCUSSION THREAD

**Discussion by aknaton** [QuestPDF/QuestPDF#406] [Q&A] [answered]
Is it possible to generate a pdf file in the PDF/A-format?

**Comment by MarcinZiabek** [comment, marked as answer]
Yes, you can enable generating PDF/A compliant document by enabling the proper flag in the `DocumentMetadata` object: https://www.questpdf.com/concepts/document-metadata#document-metadata 😁

**Reply by aknaton** [reply]
Thanks for your swift response. The question occured as I had tested this and tried to validate the PDF/A status with an online validator. That validator said it was not compliant. Maybe that was an suboptimal validator. Now I have tried to open the pdf in Acrobat Reader Pro and it says that the file claims to be PDF/A. So thanks.

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
